patterns/signature/polkadot.ts

```diff
@@ -1,4 +1,4 @@
-import { Polkadot } from "@capi/polkadot"
+import { PolkadotChain } from "@capi/polkadot"
```

Comment:
Must this be renamed? I think it's a lot nicer without the `Chain` postfix

Reply:
It would conflict with `Polkadot` (which is actually the instance of `PolkadotChainRune`).